tests/test_device_connection.py

```diff
@@ -44,23 +45,23 @@ def test_device_connection(api, utils):
     except Exception as err:
         assert err.args[0] == 500
 
-
-def test_device_lag_name(api, utils):
-    config = api.config()
-    p1 = config.ports.port(name="p1", location=utils.settings.ports[0])[-1]
-    config.options.port_options.location_preemption = True
-    lag1 = config.lags.lag(name="lag-1")[-1]
-    lp1 = lag1.ports.port(port_name=p1.name)[-1]
-    lag1.protocol.static.lag_id = 1
-    lp1.ethernet.name = "lp1-e"
-    lp1.ethernet.mac = "aa:aa:aa:aa:aa:aa"
-    d1 = config.devices.device(name="d1")[-1]
-    eth1 = d1.ethernets.ethernet()[-1]
-    eth1.connection.lag_name = lag1.name
-    eth1.name = "eth1"
-    eth1.mac = "00:01:00:00:00:02"
-    api.set_config(config)
-    assert(api._ixnetwork.Lag.find()[0].Name) == lag1.name
+# TBD: Issue assinging ports to lag
```

Comment:
better fix this issue before merge to main. As I am expecting they have this basic test.
Also better put `@pytest.mark.skip` rather than comment-out entire test. So, we will easily track it

Reply:
Fixed.